packages/react-native-codegen/src/parsers/parser.js

```diff
@@ -181,4 +183,25 @@ export interface Parser {
    * @returns: the typeParameters of the callExpression or null if it does not exist.
    */
   callExpressionTypeParameters(callExpression: $FlowFixMe): $FlowFixMe | null;
+
+  /**
+   * Given an array of properties from a Partial type, it returns an array of remaped properties.
+   * @paramater properties: properties from a Partial types.
+   * @parameter hasteModuleName: a string with the native module name.
+   * @paramater types: a map of type declarations.
+   * @paramater aliasMap: a map of type aliases.
+   * @paramater enumMap: a map of type enums.
+   * @paramater tryParse: a parser error capturer.
+   * @paramater cxxOnly: a boolean specifying if the module is Cxx only.
```

Comment:
"parameter"?.. ;)

Reply:
Ups sorry for the typo!

I see that you fixed the typo and merged the PR, thank you! :)